### ISSUE TITLE
first demo fails: No such file or directory gmaps/datasets/metadata.json

### ISSUE THREAD

**Issue by danbikle** [pbugnion/gmaps#52]
```

Here is info about my python:

(dato-env)dan@nia111:~/pg/dan $ python
Python 2.7.11 |Anaconda 2.5.0 (64-bit)| (default, Dec  6 2015, 18:08:32) 
[GCC 4.4.7 20120313 (Red Hat 4.4.7-1)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
Anaconda is brought to you by Continuum Analytics.
Please check out: http://continuum.io/thanks and https://anaconda.org
>>> 


(dato-env)dan@nia111:~/pg/dan $ jupyter --version
4.1.0
(dato-env)dan@nia111:~/pg/dan $ 

I did 

pip install gmaps
pip install ipywidgets
jupyter notebook

I see this:

import gmaps

data = gmaps.datasets.load_dataset('taxi_rides')

IOError: [Errno 2] No such file or directory: '/home/dan/anaconda3/envs/dato-env/lib/python2.7/site-packages/gmaps/datasets/metadata.json'

I am disappointed that it failed so early in the demo.

```

**Comment by danbikle** [comment]
```
work-around:
-find taxi_data.csv on github
-copy it to /tmp

data = np.genfromtxt('/tmp/taxi_data.csv', delimiter=",", names=True)

```

**Comment by pbugnion** [comment]
Yes -- this is an issue that arises with the way the package is installed when you use pip. Thanks for posting a work-around. 

I plan to fix the entire installation process, which is, as you identified, a bit broken at the moment.

**Comment by pbugnion** [comment]
Solved in v0.2.2.
